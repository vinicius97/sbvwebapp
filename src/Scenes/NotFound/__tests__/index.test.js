import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import { configure, shallow } from 'enzyme'

import { NotFound } from '../index'

configure({adapter: new Adapter()})

describe('<NotFoud />', () => {

  const wrapper = shallow(<NotFound />)

  it('Comparar snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

  it('Validar renderização', ()=> {
    expect(wrapper.find('div').text()).toEqual('Página não encontrada')
  })
})