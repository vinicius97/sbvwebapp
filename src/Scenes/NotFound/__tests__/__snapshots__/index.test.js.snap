// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoud /> Comparar snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Página não encontrada",
    },
    "ref": null,
    "rendered": "Página não encontrada",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Página não encontrada",
      },
      "ref": null,
      "rendered": "Página não encontrada",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
